### PR TITLE
Fix #7430: Update Safe Browsing setting text.

### LINKED ISSUE #7430: Update Safe Browsing setting text, follow up to #7373
See #7373

### PR DESCRIPTION
**Pull request by iccub**
## Summary of Changes

This pull request fixes #7430 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`
- [x] New or updated UI has been tested across:
  - [x] Light & dark mode
  - [x] Different size classes (iPhone, landscape, iPad)
  - [x] Different dynamic type sizes

## Screenshots:

Zrzut ekranu 2023-05-17 o 13 17 54

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by cuba — APPROVED** [review]

File: Sources/BraveShared/BraveURLs.swift [hunk 1]
```diff
@@ -24,6 +24,7 @@ extension URL {
     public static let braveVPNLinkReceiptProd = URL(string: "https://account.brave.com/?intent=connect-receipt&product=vpn")!
     public static let braveVPNLinkReceiptStaging = URL(string: "https://account.bravesoftware.com/?intent=connect-receipt&product=vpn")!
     public static let braveVPNLinkReceiptDev = URL(string: "https://account.brave.software/?intent=connect-receipt&product=vpn")!
+    public static let safeBrowsingHelp = URL(string: "https://brave.com/privacy/browser/#safe-browsing")!
```
Comment (fmarier) [inline Sources/BraveShared/BraveURLs.swift:27]: That should be https://support.brave.com/hc/en-us/articles/15222663599629-Safe-Browsing-in-Brave

**Review by fmarier — COMMENTED** [review]
Text changes look good to me, though @anthonypkeane asked for "Learn More" instead of "Learn more" in https://github.com/brave/brave-ios/issues/7374#issuecomment-1533456031

URL should point to the support page instead of the privacy policy.

**Comment by fmarier** [comment]
@iccub Sorry for not reviewing in time.

Maybe we need a quick follow up to address the above two issues (and close https://github.com/brave/brave-ios/issues/7374)?

**Comment by iccub** [comment]
I'll fix it and close the other ticket. Sorry for not waiting

**Pull request merged**